docs/source/certs_management.md

```diff
@@ -0,0 +1,381 @@
+└── peer1
+    ├── msp
+    │    ├── admincerts
+    │    │   └── cert.pem
+    │    ├── cacerts
+    │    │   └── localhost-7053.pem
+    │    ├── keystore
+    │    │   └── key.pem
+    │    ├── signcerts
+    │    │   └── cert.pem
+    │    └── user
+    |── tls
+</pre>
+
+**Impact if expired**: Production outage. Peers do not start without a valid Enrollment certificate.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-peer1)
+
+
+#### Peer TLS Certificate
+
+**Description**: Authenticates node component communication on the channel.
+
+**Location**: Dependent on implementation:
+
+<pre>
+org1ca/
+└── peer1
+    ├── msp
+    └── tls
+        ├── cacerts
+        ├── keystore
+        │   └── key.pem
+        ├── signcerts
+        │   └── cert.pem
+        ├── tlscacerts
+        │   └── tls-localhost-7053.pem
+        └── user
+</pre>
+
+**Impact if expired**: Production outage. No communication to the peer is possible.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-peer1)
+
+
+### Orderer Certificates
+
+Orderer Enrollment certificates and Orderer TLS certificates are issued for each organization.
+
+#### Orderer Enrollment Certificate
+
+**Description**: The public key that the orderer uses to sign transactions.
+
+**Location**: Dependent on implementation:
+
+<pre>
+ └── orderer1
+     ├── msp
+     │   ├── admincerts
+     │   │   └── cert.pem
+     │   ├── cacerts
+     │   │   └── localhost-7053.pem
+     │   ├── keystore
+     │   │   └── key.pem
+     │   ├── signcerts
+     │   │   └── cert.pem
+     │   |── user
+     └── tls
+</pre>
+
+**Impact if expired**: Production outage. Orderers do not start without a valid Enrollment certificate.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-orderer)
+
+
+#### Orderer TLS Certificate
+
+**Description**: TLS certificate for the ordering node.
+
+**Location**: Dependent on implementation:
+
+<pre>
+ordererca/
+└── orderer1
+    ├── msp
+    └── tls
+        ├── cacerts
+        ├── keystore
+        |   └── key.pem
+        ├── signcerts
+        │   └── cert.pem
+        ├── tlscacerts
+        │   └── tls-localhost-7053.pem
+        └── user
+  </pre>
+
+**Impact if expired**: Production outage. Ordering nodes are no longer allowed to participate in cluster.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-orderer)
+
+
+### Admin Certificates
+
+Ordering Service Organization Channel Admin certificates and Peer Service Organization Channel Admin certificates are issued for each organization.
+
+#### Ordering Service Organization Channel Admin Certificate
+
+**Description**: Admin certificate used to administer the ordering service and submit or approve channel updates. The associated admin identity is enrolled with the Organization CA before the node and Organization MSP are created.
```

Comment:
First sentence - need to indicate that the admin is associated with an organization (not the ordering service or channel as a whole).
Second sentence - I don't know what this is trying to say, you don't create an admin identity before the Organization MSP...

Reply:
First sentence: "Certificate for an organization administrator to manage ordering service and channel updates." 
Second - deleted, this would have come from a technical tester giving input.